db/people.yml

```diff
@@ -858,6 +858,33 @@ cyshi:
       group: bg
   name: Chenyang Shi
   position: scientist
+dpeters:
+  active: true
+  aka: Dom
+  avatar: https://github.com/dpeters001.png
+  bio: Dominic is a rising-senior and chemical engineering undergraduate with minors in
+  mathematics and business administration. He has experience with Linux-based and
+  python coding languages/protocols. His previous research experience includes protein
+  engineering of the SARS-CoV-2 spike protein and the mapping of energetic landscapes of 
+  viral pathogens. Dominic enjoys electronic music production, 10K races, and
+  hiking in his free time. 
+  education:
+    - begin_year: 2019
+      degree: Chemical Engineering Undergraduate
```

Comment:
Probably "BSc in Chemical Engineering"?  something like that?

Reply:
Changed to "BE Chemical Engineering"